### DISCUSSION TITLE
Inventaire integration

### DISCUSSION THREAD

**Discussion by xplosionmind** [bookwyrm-social/bookwyrm#1356] [Ideas]
**Is your feature request related to a problem? Please describe.**
It would be great if a user could:
- directly import data from their own Inventaire account
- synchronise the the books added to Inventaire and vice-versa
- Linking the same individual’s Bookwyrm and Inventaire profiles so that, until the features in #1127 are implemented, a user could make use on Inventaire of the features missing on Bookwyrm.

**Describe the solution you'd like**
Doing all of the above via the GUI without having to download, tweak in order to maximize compatibility and then re-upload exported data.

**Describe alternatives you've considered**
Doing all of the above manually.

**Additional context**
none

**Comment by mouse-reeve** [comment]
Depending on how/if Inventaire implements ActivityPub, interoperability may also been an alternative path to syncing accounts (https://github.com/inventaire/inventaire/issues/187)

**Comment by Xandru4** [comment]
I was gonna make this ticket if I hadn't found it. The scope of the two service is so close that some activity pub interoperability would be incredibly beneficial. Even more so if you could automatically sync from zotero or calibre.

**Reply by SimmelGeorg** [reply]
Yes any kind of collaboration with Zotero would be so lovely
